demo/demo_most_common.ipynb

```diff
@@ -0,0 +1,162 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demo on the \"most common\" regridder\n",
+    "\n",
+    "For regridding categorical data to a coarser resolution, normal regridding methods don't work.\n",
+    "This can be data like land cover or vegetation type.\n",
+    "\n",
+    "For this reason we implemented a \"most common\" regridder, which takes the most common value of the source data cells that overlap with the target grid cells.\n",
+    "\n",
+    "If you want to regrid data to a finer resolution, the \"nearest-neighbor\" regridder can be used."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We start with importing the required packages:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import xarray as xr\n",
+    "import xarray_regrid\n",
+    "\n",
+    "import dask.distributed\n",
+    "\n",
+    "client = dask.distributed.Client()"
```

Comment:
```suggestion
```
Not necessary since there are only some IO operations happening behind the scene (if the chunks are specified when loading the data), not anything interesting to visualize.

Reply:
I still think it's best practice to use the distributed Client, as this makes sure you use the right scheduler.